### DISCUSSION TITLE
Private Registry

### DISCUSSION THREAD

**Discussion by Nollknolle** [containrrr/watchtower#1320] [Q&A] [answered]
Hello,
i have a github organization. I am using the same credential for the org.
On my github action it works. On Watchtower not :/

I am Member on that organization and be able to connect to the repo.

Config.json
```
{
    "auths": {
        "ghcr.io": {
            "auth": "BASE64"
        }
    }
}
```

> "Unable to update container \"/angry_maxwell\": Error response from daemon: Head \"https://ghcr.io/v2/x9-studios/chipx9/manifests/74ca879c\": unauthorized. Proceeding to next."

**Comment by piksel** [comment, marked as answer]
If you run with `--debug` there should be some messages preceding that one to indicate if it found the correct credentials or not. Perhaps it would also include some more details in the response from the repo.

**Reply by Nollknolle** [reply]
Thanks for the reply.
The error was that watchtower cant read the config file.
idk why.

Now its working :)
